tests/dashboards/api_tests.py

```diff
@@ -121,14 +121,23 @@ def test_get_dashboard(self):
         db.session.delete(dashboard)
         db.session.commit()
 
+    def test_info_dashboard(self):
+        """
+            Dashboard API: Test info
+        """
```

Comment:
Still one to go

Reply:
Not one a bunch of them :) fixed all docs string for dashboards